src/features/agc/agc_content.js

```diff
@@ -507,6 +507,7 @@ function onButtonClicked() {
   }
 }
 
+
```

Comment:
Not sure why a blank line was added here but no big deal

Reply:
> Not sure why a blank line was added here but no big deal

Me neither...